src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
@lemire This is indeed very useful and can lead to techniques to use not just SSE/AVX, but heterogeneous hardware as well. We will be happy to assist you in building your prototype too.